app/src/main/java/com/fastlink/zekrapp/presentation/utils/bottomAppBar/homeAndFavoriteBottomBar.kt

```diff
@@ -12,43 +12,36 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
-import androidx.navigation.NavBackStackEntry
-import androidx.navigation.NavController
+
 import androidx.navigation.compose.currentBackStackEntryAsState
 import com.fastlink.zekrapp.LocalNavController
+import com.fastlink.zekrapp.R
 import com.fastlink.zekrapp.presentation.navigation.Screen
 
 @Composable
-fun BottomBar() {
+fun BottomAppBarForHomeAndFavoriteScreens(
```

Comment:
Choose a better name, don't mention where the component is used, suppose you changed your mind and wanted to use the component in a different screen, are you going to change the name and add another "and"?

Reply:
you are right, but we have 2 different BottomAppBars we can't say BottomAppBar1 And BottomAppBar2 for that, 
we have a only 3 screens if we had another screen I didn't change the name this  i used the same previous name `BottomBar`

Reply:
Either delete one of them and make the bottom bar reusable for all the screens, or choose a better names for them. If we have 2 bottom bars that doesn't mean to name them like that.